### DISCUSSION TITLE
[Question] Which fonts can I use?

### DISCUSSION THREAD

**Discussion by DarthWotan** [Saadmairaj/tkmacosx#31] [Q&A]
Hey, 
I am making a little gui with Tkmacox and I wanted to ask if there is any option to see all available fonts? I searched in the internet but I don't know if the Tkinter fonts are also the Tkmacosx fonts or if there are more. 
Thank you @Saadmairaj ^_^

**Comment by Saadmairaj** [comment]
Hi,
As Tkmacosx has been build with Tkinter, all the fonts that work with Tkinter should also work with Tkmacosx. You can checkout this[ post at stackoverflow](https://stackoverflow.com/questions/39614027/list-available-font-families-in-tkinter) to see and understand all the fonts that works with Tkinter. Also, Tkinter itself uses the fonts available in the system, so if you want you can install any font into your system which then can be used in Tkinter and Tkmacosx as well.
